packages/core/database/lib/entity-manager/regular-relations.js

```diff
@@ -296,7 +296,9 @@ const cleanOrderColumnsForInnoDB = async ({
                 FROM :joinTableName: b
                 WHERE a.:orderColumnName: >= b.:orderColumnName: AND a.:joinColumnName: = b.:joinColumnName: AND a.:joinColumnName: = :id
               ) AS src_order
-            FROM :joinTableName: a`,
+            FROM :joinTableName: a
+            WHERE a.:joinColumnName: = :id
+            `,
```

Comment:
Nice :)

Is it also needed line 343?

Also, can we write some tests about that?

Reply:
absolutely :) , done! I have not done an exhaustive test to look if everything is working on this last change. But relations , compos and dz seems to work ok.